src/main/java/com/vntu/marenko/ualearning/server/config/CorsConfig.java

```diff
@@ -0,0 +1,20 @@
+package com.vntu.marenko.ualearning.server.config;
```

Comment:
Please delete this file. It is obsolete, as this configuration not working. I believe it remained in your repo as old code. You should use git pull -r {remote_name} {branch name} command to download the latest version, especially before making the pull/merge requests